01-JS-DOM-APIs/js/table.js

```diff
@@ -0,0 +1,23 @@
+// arr must be bidimensional
+function makeTable(arr) {
+  var tableNode = document.createElement("table");
+  document.getElementById("table-div").appendChild(tableNode);
+
+  for (var r = 0; r < arr.length; r++) {
```

Comment:
It is better to use map to map an array. Here you could find documentation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map